ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/tcam-profile.j2

```diff
@@ -1,4 +1,5 @@
-{% if tcam_profile is defined and tcam_profile is not none %}
+{# eos - tcam profile #}
+{% if tcam_profile is arista.avd.defined %}
 !
 hardware tcam
    system profile {{ tcam_profile[0] }}
```

Comment:
missing check for tcam_profile item 0. Are we sure this is what we want? It looks strange.

Reply:
This is a list of only one element. I propose to use ``` | first``` to just take the first configured list (only one hw profile allowed). From my point of view, better solution will be to have an index like name for the value, but this will be a breaking change

Reply:
I think @titom73 is working on a complete reimplementation of tcam profiles, so we can ask him to include a datamodel change here.

Reply:
Indeed, a PR has been submitted and will be refactored to support more on TCAM profile (https://github.com/aristanetworks/ansible-avd/pull/622)

Expected data model would be:

```yaml
tcam_profile:
  system: < tcam profile name to activate en EOS device >
  profiles:
    < tcam_profile 01 >: < tcam profile in EOS syntax >
```

So you can just let it has it is today since 2.0.0rc3 will have an update for TCAM (and feel free to share your thoughts for the next model version)